docs/Configuring-Pantheon/Networking.md

```diff
@@ -5,7 +5,21 @@ description: Pantheon networking is about P2P discovery and communication betwee
 
 Pantheon uses the network to find and connect to peers. 
 
-## Firewalls and Incoming Connections
+## Port Configuration 
```

Comment:
I think describing this as Incoming Firewall configuration is clearer.

Reply:
Leaving as port configuration as discussed (relevant to when running in Docker as well as firewalls)